diff -up goffice-0.10.12/mmlitex/Makefile.am.destdir goffice-0.10.12/mmlitex/Makefile.am
--- goffice-0.10.12/mmlitex/Makefile.am.destdir	2014-02-23 23:55:00.000000000 +0100
+++ goffice-0.10.12/mmlitex/Makefile.am	2014-03-04 21:21:12.603827954 +0100
@@ -7,6 +7,6 @@ xslt_DATA = \
 	mmlitex.xsl	\
 	tables.xsl \
 	README 
-xsltdir = $(DESTDIR)$(goffice_datadir)/mmlitex
+xsltdir = $(goffice_datadir)/mmlitex
 
 EXTRA_DIST = $(xslt_DATA)
diff -up goffice-0.10.12/mmlitex/Makefile.in.destdir goffice-0.10.12/mmlitex/Makefile.in
--- goffice-0.10.12/mmlitex/Makefile.in.destdir	2014-03-04 00:43:14.000000000 +0100
+++ goffice-0.10.12/mmlitex/Makefile.in	2014-03-04 21:21:12.604827963 +0100
@@ -136,7 +136,7 @@ am__uninstall_files_from_dir = { \
     || { echo " ( cd '$$dir' && rm -f" $$files ")"; \
          $(am__cd) "$$dir" && rm -f $$files; }; \
   }
-am__installdirs = "$(DESTDIR)$(xsltdir)"
+am__installdirs = "$(xsltdir)"
 DATA = $(xslt_DATA)
 am__tagged_files = $(HEADERS) $(SOURCES) $(TAGS_FILES) $(LISP)
 DISTFILES = $(DIST_COMMON) $(DIST_SOURCES) $(TEXINFOS) $(EXTRA_DIST)
@@ -398,23 +398,23 @@ install-xsltDATA: $(xslt_DATA)
 	@$(NORMAL_INSTALL)
 	@list='$(xslt_DATA)'; test -n "$(xsltdir)" || list=; \
 	if test -n "$$list"; then \
-	  echo " $(MKDIR_P) '$(DESTDIR)$(xsltdir)'"; \
-	  $(MKDIR_P) "$(DESTDIR)$(xsltdir)" || exit 1; \
+	  echo " $(MKDIR_P) '$(xsltdir)'"; \
+	  $(MKDIR_P) "$(xsltdir)" || exit 1; \
 	fi; \
 	for p in $$list; do \
 	  if test -f "$$p"; then d=; else d="$(srcdir)/"; fi; \
 	  echo "$$d$$p"; \
 	done | $(am__base_list) | \
 	while read files; do \
-	  echo " $(INSTALL_DATA) $$files '$(DESTDIR)$(xsltdir)'"; \
-	  $(INSTALL_DATA) $$files "$(DESTDIR)$(xsltdir)" || exit $$?; \
+	  echo " $(INSTALL_DATA) $$files '$(xsltdir)'"; \
+	  $(INSTALL_DATA) $$files "$(xsltdir)" || exit $$?; \
 	done
 
 uninstall-xsltDATA:
 	@$(NORMAL_UNINSTALL)
 	@list='$(xslt_DATA)'; test -n "$(xsltdir)" || list=; \
 	files=`for p in $$list; do echo $$p; done | sed -e 's|^.*/||'`; \
-	dir='$(DESTDIR)$(xsltdir)'; $(am__uninstall_files_from_dir)
+	dir='$(xsltdir)'; $(am__uninstall_files_from_dir)
 tags TAGS:
 
 ctags CTAGS:
@@ -456,7 +456,7 @@ check-am: all-am
 check: check-am
 all-am: Makefile $(DATA)
 installdirs:
-	for dir in "$(DESTDIR)$(xsltdir)"; do \
+	for dir in "$(xsltdir)"; do \
 	  test -z "$$dir" || $(MKDIR_P) "$$dir"; \
 	done
 install: install-am
